### ISSUE TITLE
NTR: Borrelia miyamotoi PCR positive (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8112]
New term request Borrelia miyamotoi PCR positive (segal_200720134126): Polymerase chain reaction; DNA

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000907
- label: Positive bloodstream Borrelia miyamotoi nucleic acid test
- parents: [Positive bloodstream nucleic acid pathogen test (HP:0430067)](https://hpo.jax.org/app/browse/term/HP:0430067)
- pmid: PMID:29664394
- def: Detection of nucleic acid of Borrelia miyamotoi in the blood circulation by a method such as polymerase chain reaction (PCR).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8112
- synonyms:
